simtools/corsika_simtel/corsika_simtel_runner.py

```diff
@@ -136,34 +136,30 @@ def _make_run_command(self, **kwargs):
 
         """
 
-        # TODO: These definitions of the files can probably be separated from
-        # the the run command and put back into the parent class.
         info_for_file_name = SimtelRunnerArray.get_info_for_file_name(self, kwargs["run_number"])
-        self._log_file = SimtelRunnerArray.get_file_name(
-            self, file_type="log", **info_for_file_name
-        )
-        histogram_file = SimtelRunnerArray.get_file_name(
-            self, file_type="histogram", **info_for_file_name
-        )
-        output_file = SimtelRunnerArray.get_file_name(
-            self, file_type="output", **info_for_file_name
-        )
+        weak_pointing = any(pointing in self.label for pointing in ["divergent", "convergent"])
```

Comment:
That's how Konrad called it
```
#   weak_pointing: The telescope_theta and telescope_phi setting with 'weak'
#                 configuration option (-W) which can be overridden in config files.
#                 That is mainly for explicit convergent/divergent pointing.
```
It makes sense though because otherwise when we pass along the zenith/azimuth parameters with `-C` it overrides the pointings in the config files. When passing it with `-W`, the pointing will be passed along to the telescopes only if it is not set in the config files, hence, "weak" option.

Reply:
OK - it is simply that I've seen this the first time. All good!